README.md

```diff
@@ -63,7 +63,7 @@ For more detailed information on setting these up, see ["Advanced setup"](#advan
 
 ## Requirements
 
-- **Neovim 0.8.0 or later** built with **tree-sitter 0.20.3+** (latest [nightly](https://github.com/neovim/neovim#install-from-source) recommended)
+- **Neovim 0.8.3 or later** built with **tree-sitter 0.20.3+** (latest [nightly](https://github.com/neovim/neovim#install-from-source) recommended)
```

Comment:
Can't we make that statement more generic? Like `latest Neovim release`? I feel that otherwise this can get outdated pretty fast.

Reply:
maybe with hyperlink to https://github.com/neovim/neovim/releases/tag/stable

Reply:
> I feel that otherwise this can get outdated pretty fast.

@theHamsta, sorry for the delay! 

it's generally useful to know which release is required, but I guess we can make an exception in this case that it's always following the latest.